<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1038</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-5</y>
      <width>1291</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="logolabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logout_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Log out</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addbook_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add Book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewbook_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>View Books</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addmember_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add Member</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewmember_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>View Members</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="overduebooks_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Overdue Books</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkoutbooks_button">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Checkout Books</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="activeLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>500</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
